packages/core/src/odata-common/request/odata-request.spec.ts

```diff
@@ -50,6 +52,7 @@ describe('OData Request', () => {
           ODataDeleteRequestConfig,
           destination
         );
+        request.config.fetchCsrfToken = false;
```

Comment:
In your initial description you mention something about disabling the csrf handling. This would mean to add a method like `disableCSRF` on the request builder because the line above is just a work around right?

Reply:
I want to make a new PR for a new API of the request builder, which will then control the `request.config.fetchCsrfToken`.

Reply:
Just FMI: Why are we setting this to false here?

Reply:
See [line 51](https://github.com/SAP/cloud-sdk-js/blob/migrate-odata-csrf/packages/core/src/odata-common/request/odata-request.spec.ts#L51), where it calls `createRequestWithHeaders`.
The `createRequestWithHeaders` contains a mock:
```
jest.spyOn(req, 'headers').mockResolvedValue({});
```
This mock fits the old implementation and skips the csrf token fetching.
Now we migrate to the new version, where the `http-client` holds the logic of fetching the token.
I would like to keep using the same behaviour (do not consider csrf token) and make it more explicit.